internal/io/sql/coerce.go

```diff
@@ -26,20 +25,3 @@ func Int64ToBool(c *Column) func(t interface{}) error {
 		return nil
 	}
 }
-
```

Comment:
This does not seem quite right. You're reverting the commits from the previous PR here. Same thing in `sql/config.go`.